runtime/bcutil/bcutil.c

```diff
@@ -134,7 +134,7 @@ bcutil_J9VMDllMain(J9JavaVM* vm, IDATA stage, void* reserved)
 		/* Note that verbose support should have already been initialized in an earlier stage */
 		case BUFFERS_ALLOC_STAGE :
 			loadInfo = FIND_DLL_TABLE_ENTRY( THIS_DLL_NAME );
-			if (J2SE_VERSION(vm) >= J2SE_V11) {
+			if (JAVA_SPEC_VERSION >= J2SE_V11) {
```

Comment:
Have you verified that no compiler issues a warning (that would be considered an err) about what will become a constant `if` condition?

Reply:
Yeah, it was verified via personal build at platform `Linux AMD`, `Windows` and `zOS`.